### PR TITLE
nvidia: fix GHSA-m425-mq94-257g

### PR DESCRIPTION
**Pull request by hectorj2f**
```
wolfictl scan packages/aarch64/nvidia-device-plugin-0.14.2-r2.apk
🔎 Scanning "packages/aarch64/nvidia-device-plugin-0.14.2-r2.apk"
✅ No vulnerabilities found
```

### CODE REVIEW

File: nvidia-device-plugin.yaml [hunk 1]
```diff
@@ -38,6 +41,7 @@ pipeline:
 
 update:
   enabled: true
+  manual: false
```
Comment (luhring) [inline nvidia-device-plugin.yaml:44]: Why this change? Just curious

Comment (hectorj2f) [reply nvidia-device-plugin.yaml:44]: This is a default value, honestly I don't see a problem to keep it. But I could change my script.

Comment (hectorj2f) [reply nvidia-device-plugin.yaml:44]: My code is also re-ordering the properties according to the initial yaml structure.

Comment (luhring) [reply nvidia-device-plugin.yaml:44]: AFAIK we normally don't have `manual: false` in these files

Comment (hectorj2f) [reply nvidia-device-plugin.yaml:44]: @luhring Almost all the py3- files set it to false. https://github.com/wolfi-dev/os/pull/6863/files#diff-5fbae266c308c5a4453523df15a67dabe1e6a8e8770be85ea47632df4534be66R57 We have many more files with this property and values true | false.

**Review by luhring — APPROVED** [review]
